miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/Displayer.kt

```diff
@@ -1,21 +1,18 @@
 package com.rakuten.tech.mobile.miniapp.display
 
-import android.content.Context
 import com.rakuten.tech.mobile.miniapp.MiniAppDisplay
 import com.rakuten.tech.mobile.miniapp.js.MiniAppMessageBridge
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 
-internal class Displayer(private val context: Context) {
+internal class Displayer {
 
     suspend fun createMiniAppDisplay(
         basePath: String,
         appId: String,
         miniAppMessageBridge: MiniAppMessageBridge
     ): MiniAppDisplay =
         withContext(Dispatchers.Main) {
```

Comment:
Because not creating webview so does not need to put in this looper anymore. Just only:
```suggestion
        context?.let {
                RealMiniAppDisplay(basePath, appId, miniAppMessageBridge)
            }
```